lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -90,6 +90,18 @@ class ViaqDataModelFilter < Filter
     desc 'Name of destination timestamp field'
     config_param :dest_time_name, :string, default: '@timestamp'
 
+    # <event_trans>
+    #   type eventrouter
+    #   tag "kubernetes.**"
+    # </event_trans>
+    desc 'Perform event transformations'
+    config_section :event_trans, param_name: :event_trans do
+      desc 'Process records with this tag pattern'
+      config_param :tag, :string
+      desc 'Type of event transformation'
+      config_param :type, :enum, list: [:none, :eventrouter]
```

Comment:
What is the purpose of having a `:none` type?

Reply:
I had it here as a placeholder for docker event transformations